sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -320,27 +320,31 @@ TEST(AzureCliCredential, CmdLine)
 TEST(AzureCliCredential, UnsafeChars)
 {
   std::string const Exploit = std::string("\" | echo OWNED | ") + InfiniteCommand + " | echo \"";
+  std::vector<std::string> exploits{Exploit, "space character"};
 
+  for (std::string exploit : exploits)
```

Comment:
Please copy&paste a separate testcase for the space character - loops in unit tests do not produce clear results.

Reply:
+1 to this. There have been many situations where I've had to break apart test cases to isolate individual failures.